lib/galaxy/util/script.py

```diff
@@ -11,6 +11,19 @@
 ARGUMENTS = None
 DEFAULT_ACTION = None
 
+ARG_HELP_CONFIG_FILE = """
+Galaxy config file (defaults to $GALAXY_ROOT/config/galaxy.yml if that file exists
```

Comment:
This would be more accurately `./config/galaxy.yml`

Reply:
I guess it is - I'm not sure how I feel about that. I'll update the docs though.